src/main/java/Storage.java

```diff
@@ -0,0 +1,86 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Storage {
+
+    public static File createFile(String filepath) {
+        try {
+            File f = new File(filepath);
+            if (f.createNewFile()) {
+                System.out.println("File created: " + f.getName());
+            } else {
+            }
```

Comment:
please do not have empty catch blocks. if you choose, you may just check for the condition in a guard clause